src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -787,15 +787,16 @@ def _validate_vm_vmss_accelerated_networking(cli_ctx, namespace):
                 return
 
         # VMs need to be a supported image in the marketplace
-        # Ubuntu 16.04 | 18.04, SLES 12 SP3, RHEL 7.4, CentOS 7.4, CoreOS Linux, Debian "Stretch" with backports kernel
+        # Ubuntu 16.04 | 18.04, SLES 12 SP3, RHEL 7.4, CentOS 7.4, Flatcar Linux, Debian "Stretch" with backports kernel
```

Comment:
We should use "Flatcar Container Linux" (or just "Flatcar" for short), but not "Flatcar Linux".

Reply:
OK, done